geodata/templates/geodata/fmventry_detail.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+{% block content %}
+  <article class="media content-section">
+    <div class="media-body">
+      <div class="article-metadata">
+        <small class="text-muted">Created: {{ object.created }}</small>
+        <p/>
+        <small class="text-muted">Modified: {{ object.modified }}</small>
+      </div>
+
+      <h2 class="article-title">
+        <span>{{ object.name }} ({{object.pk}})</span>
+      </h2>
+      <hr/>
+
+      <p class="article-content">Description: {{ object.description }}</p>
+
+      <video width="320" height="240" autobuffer="autobuffer" autoplay="autoplay" loop="loop" controls="controls">
+        <source src="{{ object.fmv_file.file.url }}" type='video/mp4; codecs="avc1.42E01E, mp4a.40.2"'>
+      </video>
```

Comment:
This is giving me the correct URL to the video source (I can inspect the page and download the video from that link). But it isn't playing (videos are in `.mpg` format)

Reply:
So turns out these VIRAT videos are in `MPEG-TS` format which we may need special tooling to play in a browser. https://github.com/video-dev/hls.js/ may be able to handle transcoding the video